addons/l10n_mx/data/template/account.tax-mx.csv

```diff
@@ -1,53 +1,53 @@
-"id","sequence","name","description","invoice_label","amount","amount_type","type_tax_use","tax_group_id","tax_exigibility","cash_basis_transition_account_id","l10n_mx_tax_type","active","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/tag_ids","name@es"

```

Comment:
Since name of the tax have to be a little code, we decided to remove it and put the translation only on the description

Reply:
But does it make sense ? If I'm a Spanish speaker but do not speak English, when I see "20% WH", I don't not what it is. It should be "20% Ret" or something like that  in Spanish.

Reply:
We talked to CHKL and thought that people will learn the code, at first they will need to see the description but once they are used to it, it will not be a problem